sources/osgGA/SwitchManipulator.js

```diff
@@ -77,8 +76,11 @@ define( [], function () {
                 return manipulator.getInverseMatrix();
             }
         },
-        setByMatrix: function ( matrix ) {
```

Comment:
Because it's not used.
I added it when I did the compute home position projection stuffs.
But I didn't implemented the function in the orbit and fps manipulator and I forgot to remove it.